x-pack/plugins/ml/public/application/jobs/new_job/utils/new_job_utils.test.ts

```diff
@@ -140,24 +140,50 @@ describe('createSearchItems', () => {
     savedSearch.attributes.kibanaSavedObjectMeta.searchSourceJSON = JSON.stringify(searchSource);
 
     const resp = createSearchItems(kibanaConfig, indexPattern, savedSearch);
-    expect(resp).toStrictEqual({
-      combinedQuery: {
-        bool: {
-          filter: [
-            { bool: { should: [{ match_phrase: { airline: 'AAL' } }], minimum_should_match: 1 } },
-            {
-              bool: {
-                must_not: {
-                  bool: { should: [{ match_phrase: { airline: 'AWE' } }], minimum_should_match: 1 },
+    expect(resp).toMatchInlineSnapshot(`
```

Comment:
Are you changing these so that way it's just easier to jest update them in the future with the jest update command?

I think this is fine for these here and a good idea.